includes/SVG.php

```diff
@@ -105,6 +105,7 @@ public function init() {
 
 		// Check if svg uploads, already enabled.
 		if ( $this->svg_already_enabled() ) {
+			add_filter( 'mime_types', [ $this, 'mime_types_add_svg' ] );
```

Comment:
Basically, we are adding this to make sure that mime_type is in this list, safe svg does not add it, it only adds it to upload_types. So just add it to be safe and if it is already there, the array unique will remove the duplicate.

Reply:
Now we're only calling `add_filter( 'mime_types', [ $this, 'mime_types_add_svg' ] );` if Safe SVG is active, but not when it is absent.

So if Safe SVG is absent, mime type list will not be filtered. Is that expected?

Reply:
So if `svg_already_enabled` is true then we add mime_types just to be sure. We can't add it before the check, because it would make the check always pass. This is an ordering issue, which is why the test were before. 

This line is still place. 
https://github.com/google/web-stories-wp/blob/ea1da0345f3e87f0092645b0c35dc0fdd0f8661d/includes/SVG.php#L111

Reply:
I guess I am confused then about the purpose of `upload_mimes` vs `mime_types`

Reply:
I think it is confusing, as it is confusing in core. From what I can workout, `upload_mimes` are just support mime types you are allowed to upload. Adding to `mime_types` also seems to also add it to [get_allowed_mime_types](https://github.com/WordPress/wordpress-develop/blob/176a1f53f04cde92e7297b5214c03beb9e2ba5c8/src/wp-includes/functions.php#L3241-L3262) function. 

For some reason safe svg doesn't add to `mime_types` just some reason `upload_mimes`.  The reason it is adding after the check on `svg_already_enabled`, adding before, will make the `svg_already_enabled` check always return as true. 

TDLR, safe svg is doing something weird, this is a workaround for it.